### DISCUSSION TITLE
Guilded Style Discord Server Imports

### DISCUSSION THREAD

**Discussion by MavisCelus19201** [stoatchat/discussions#305] [Feature Suggestions]
On Guilded one thing you can do is import server templates from Discord via that services template export, and then import them to get the channels, roles and that sort.

Maybe something similar for Revolt would be good for ease of migration. Only thing Guildeds method doesn't cover is messages, not sure how possible *those* would be though programs to back those up like DHE exist at the very least.

**Comment by IdotMaster1** [comment]
We need this! It would really help people make the switch.
